### DISCUSSION TITLE
v0.2.1

### DISCUSSION THREAD

**Discussion by jrestall** [jrestall/remix-kit#19] [Announcements]
:rocket v0.2.1 has been released on npm. 

There's a breaking change,
- @remix-kit/cli package is now remix-kit. Please update your dependencies.

This release enables a much smoother RemixKit install with a npx command to handle everything `npx remix-kit@latest install`. Please see the updated [getting started guide](https://github.com/jrestall/remix-kit/blob/main/docs/getting_started.md). You can now introduce Vite & HMR to your project alongside the existing Remix dev experience in under a minute, with no or minor changes to your code.

The getting started guide is much smaller and simpler, with no need to edit any server or root.tsx files. ReactRefresh is automatically injected and the CSS ?url suffix is handled for you.

**Comment by cliffordfajardo** [comment]
Thanks for the update! Needing to configure less things out of the box is great!

I ran into some challenges with the version `0.2.1`, maybe I did something wrong though; I outlined my reproduction in the link below:
- https://github.com/jrestall/remix-kit/issues/20
